firebase-firestore/src/main/java/com/google/firebase/firestore/model/MutableDocument.java

```diff
@@ -19,12 +19,13 @@
 
 /**
  * Represents a document in Firestore with a key, version, data and whether it has local mutations
- * applied to it. Documents can start out as {@link DocumentType#INVALID} and transition to a valid
- * states via {@link #convertToFoundDocument(SnapshotVersion,ObjectValue)}, {@link
- * #convertToNoDocument(SnapshotVersion)} and {@link #convertToUnknownDocument(SnapshotVersion)}.
+ * applied to it.
  *
- * <p>Invalid documents serve as base documents for mutations. If a document remains invalid even
- * after all mutations have been applied, it should be removed from all views.
+ * <p>Documents can transition between states via {@link
+ * #convertToFoundDocument(SnapshotVersion,ObjectValue)}, {@link
```

Comment:
Optional: For maintainability and brevity of this documentation, you can omit the arguments in the `@link` tags. For example,  `{@link #convertToFoundDocument(SnapshotVersion,ObjectValue)}` can simply be `{@link #convertToFoundDocument}`. Both javadoc generation and IDE code navigation will infer the correct arguments.

Reply:
Interesting. I thought that is how it worked, but I couldn't get the linking to work. It turns out it only works without `()`.